gulpfile.js

```diff
@@ -78,4 +78,4 @@ gulp.task('watch', function() {
 });
 
 gulp.task('styles', ['styles:dev', 'styles:prd','stylesAdmin']);
-gulp.task('default', ['styles', 'watch']);
+gulp.task('default', ['styles','compress']);
```

Comment:
Why the change here? I don't see how this is related to the loader? Just curious..

Reply:
The JS is not included in the default tasks, will updated ['styles','compress','watch']